File: Ass_14/Assignment_2.c
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0, j = 0;
    int even = 2;
    int odd = 1;

    for(i = 1; i <= iRow; i++)
    {
        if(i % 2 != 0) 
        {
    
            for(j = 1; j <= iCol; j++)
            {
                printf("%d\t", even);
                even = even +  2;
            }
        }
        else 
        {
            for(j = 1; j <= iCol; j++)
            {
                printf("%d\t", odd);
                odd = odd +  2;
            }
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows: ");
    scanf("%d", &iValue1);

    printf("Enter number of columns: ");
    scanf("%d", &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: Ass_47/Assignment_2.java
import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
        int i = 0, j = 0;

        for(i = 0;i <iRow;i++)
        {
            for(j = 1; j <iCol;j++)
            {
                System.out.print(j+" ");
                
            }
            System.out.println("\n");

        }
        
    }
}
class Assignment_2
{
    public static void main(String[]args)
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter the number of columns :");
        int iCol = sobj.nextInt(); 

        Pattern pobj = new Pattern();
        pobj.Pattern(iRow,iCol);
    }
}